modules/gapi/src/backends/plaidml/gplaidmlbackend.cpp

```diff
@@ -50,13 +51,23 @@ namespace
             gm.metadata(op_node).set(cv::gimpl::PlaidMLUnit{plaidml_impl});
         }
 
-        virtual EPtr compile(const ade::Graph &graph,
-                             const cv::GCompileArgs &,
-                             const std::vector<ade::NodeHandle> &nodes,
+        virtual EPtr compile(const ade::Graph& graph,
+                             const cv::GCompileArgs& args,
+                             const std::vector<ade::NodeHandle>& nodes,
                              const std::vector<cv::gimpl::Data>& ins_data,
                              const std::vector<cv::gimpl::Data>& outs_data) const override
         {
-            return EPtr{new cv::gimpl::GPlaidMLExecutable(graph, nodes, ins_data, outs_data)};
+            auto has_config = cv::gimpl::getCompileArg<cv::gapi::plaidml::config>(args);
+
+            if (!has_config)
+            {
+                cv::util::throw_error(std::runtime_error("Config not found!\n"
+                                                         "You must pass cv::gapi::plaidml::config to the graph compile arguments"));
+            }
```

Comment:
Why it is mandatory? .plaidml file doesn't work anymore?

Reply:
In the new API this is passed explicitly. I have to ask plaidml team about it.